### ISSUE TITLE
[ROS 2] Error messages if action server exists

### ISSUE THREAD

**Issue by jacobperron** [ros-visualization/rqt_service_caller#14]
If there is an action server on the ROS graph, when rqt_service_caller starts we see errors message about invalid service types.

To reproduce, start turtlesim

    ros2 run turtlesim turtlesim_node

And start rqt_service_caller:

    ros2 run rqt_service_caller rqt_service_caller 

Example error message:

```
[ERROR] [1589841268.191775247] [get_service_class]: Malformed srv message_type: turtlesim/action/RotateAbsolute_GetResult
ServiceCaller.on_refresh_services_button_clicked(): could not get class turtlesim/action/RotateAbsolute_GetResult of service /turtle1/rotate_absolute/_action/get_result. Skipping it.
[ERROR] [1589841268.192278390] [get_service_class]: Malformed srv message_type: turtlesim/action/RotateAbsolute_SendGoal
ServiceCaller.on_refresh_services_button_clicked(): could not get class turtlesim/action/RotateAbsolute_SendGoal of service /turtle1/rotate_absolute/_action/send_goal. Skipping it.
```

I don't necessarily expect the service caller to offer support for calling these "hidden" services, but it should know enough to ignore them without producing errors.

**Comment by wjwwood** [comment]
I proposed https://github.com/ros-visualization/rqt_service_caller/pull/20 to fix this.

**Comment by wjwwood** [comment]
Closed with #20
